src/Utils.ts

```diff
@@ -10,6 +11,15 @@ class Utils {
   static round(number: number): number {
     return Math.round((number + Number.EPSILON) * 100) / 100;
   }
+
+  // Generate short id without collision checking
+  // Shouuld be ok for < 1K  collection
+  // No need for secure id (crypto)
+  static generateUID(): string {
+    const firstPart = (`000${((Math.random() * 46656) | 0).toString(36)}`).slice(-3);
+    const secondPart = (`000${((Math.random() * 46656) | 0).toString(36)}`).slice(-3);
```

Comment:
Just a comment, not asking for a change unless you want to, open to discussion.

I'm completely OK with using | 0, but I've had a discussion about this a few months ago; instead of using | 0, I've used ~~ to round a number. Now, we know what ~~number and number | 0 does, and I was totally fine using that, but @tylp made an interesting remark: shouldn't we try to make the code clear to everyone? I'd suggest either wrapping the rounding in a:
```ts
class Utils {
  static round(n: number): number {
    return n | 0;
  }
}
```
Although I do see that we already have a round function

Or we could use Math.round(Math.random() * 46656)), thus being clear from the get go. I've checked the performances, over an array of 10 000 000 random numbers. Here are the results:
![image](https://user-images.githubusercontent.com/22148893/141107773-a1e10665-97c6-4df0-b59c-ad259e69f03c.png)

We could perfectly use Math.round without any performance issue, I reckon.

Reply:
The code comes from https://stackoverflow.com/a/6248722/13885834
I assume that this code will not be readable anyway.
Do we really want this code to be readable and to understand the internal temblor? I don't know. We just want to know that it generates an ID. 
I don't fully understand this code and I don't really need it because it's just a utility function.

Reply:
Alright, fine by me!